lib/covenants/rules.js

```diff
@@ -572,6 +582,19 @@ rules.countUpdates = function countUpdates(tx) {
   return total;
 };
 
+/**
+ * Check whether this covenant counts against the update limit.
+ * @param {String} type
+ * @returns {Boolean}
+ */
+const updateLimitedTypes = new Set(types.OPEN, types.UPDATE, types.TRANSFER, types.REVOKE);
+
+rules.isUpdateLimited = function isUpdateLimited(typeStr) {
```

Comment:
I think this should take a `Number` instead of a string

Reply:
And we should be very sure that this function doesn't already exist in the codebase someplace